packages/salesforcedx-utils-vscode/test/jest/cli/forceConfigGet.test.ts

```diff
@@ -0,0 +1,117 @@
+import { ForceConfigGet } from '../../../src';
```

Comment:
The existing unigration tests were being dumb so I just converted to jest unit tests.